src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -4834,7 +4832,6 @@ private bool IsUsingAliasInScope(string name)
                 return BadExpression(node, boundLeft);
             }
 
-            // PROTOTYPE(default) unify with case above
             // No member accesses on default
             if (boundLeft.IsLiteralDefault())
```

Comment:
I wonder if these two cases (default and unbound lambda) can be combined by testingfor `(object)leftType == null` and in that case report the diagnostic using `boundLeft.Display`.